drive/client.bal

```diff
@@ -462,12 +462,12 @@ public client class Client {
     # + pageToken - The page token  
     # + expiration - The expiration time
     # + return - If successful, returns `WatchResponse`. Else returns `error`
-    @display {label: "Watch specific using file id"} 
+    @display {label: "Watch Specific Using File Id"} 
```

Comment:
What is `WatchResponse `? would not `Get Watch Response` will work?

Reply:
`WatchResponse` is the response we receive after subscribing to a watch. Currently, we have hidden these watch methods from choreo. By the way, we will send another PR after moving these methods to the listener.